.github/workflows/dev.yml

```
@@ -9,7 +9,17 @@ on:
       - 'docs/**'
       - '**/README.md'
 jobs:
+  set_version:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    outputs:
      VERSION: ${{ steps.version.outputs.VERSION }}
```

I'm not sure if this map is **required**, but it is [documented here](https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs#example-defining-outputs-for-a-job).

---

thanks!